### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by eNaimi-dt3cc** [Chia-Network/chia-blockchain#11687] [General]
### What happened?

Prior to Chia version 1.3, all plots were counted for each Chia pool I connected to, and I received chia currency for that, but after version 1.3, only a quarter of the plots in each pool I connected to. Pools are calculated, and the amount of daily and monthly receipts is a quarter, for example, I used to receive about 0.3 chia per month for 500 plots, but now I receive about 0.08 chia for 500 plots, and this amount for several pools I checked

### Version

1.3.5

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by shaneo257** [comment]
have you reached out to the pool you connect to?

**Reply by eNaimi-dt3cc** [reply]
At first, I was connected to the xchpool for a few months
And with 500 plots per month, I received about 0.3 chia
After the arrival of version 1.3, I received 500 plots
It reached about 0.08 Chia per month
I tried the teepool pool, it was the same
I am also checking the flexpool pool now
Again, my intake has dropped from 0.3 chia per month to about 0.08 chia
Interestingly, in the flexpool pool, it is sometimes said that the space of your plots is 30 tib
And sometimes, for example, it says 6 tib
And the average space of my plots is above 50 tib
11 tib announces
And compared to the same average of 11 tibs per month, it pays about 0.08 chia

Meanwhile, I have been connected to Chia pools for about 10 months

#ChiaPlots #Rewards #ChiaPools #ChiaCoin

![Screenshot (1)](https://user-images.githubusercontent.com/106408380/170819005-c0c9d0dc-e9e1-4fab-b45b-eb5817b84ef3.png)
![Screenshot (3)](https://user-images.githubusercontent.com/106408380/170819013-26239165-3d2a-46e0-8918-4814e20d186f.png)
a
